onboarding/helpers.py

```diff
@@ -1,14 +1,31 @@
+import re
 import pytz
 from logging import getLogger
 
 from dateutil.relativedelta import relativedelta
 from datetime import datetime
+from difflib import SequenceMatcher
+import pycountry
 
+from django.db.models.functions import Length
+
+from onboarding.constants import ORG_SEARCH_TERM_LENGTH
+from onboarding.models import Organization
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 utc = pytz.UTC
 log = getLogger(__name__)
 
 
+def get_country_name(country_code):
+    try:
+        return pycountry.countries.get(alpha_2=country_code).name
+    except AttributeError:
+        log.error(f"Invalid country code: {country_code}")
```

Comment:
It is not so informative.

```suggestion
        log.exception(f"Can't resolve the {country_code!r} country code into the country name.")
```

Notes:
1. Log type changed to exception to include the exec_info
2. string format is applied to get the country code in quotes in the final log message - THIS SHOULD BE TESTED with Python 3.8 because it could be a feature of the newer version of Python (I'm not sure)

Reply:
`!r` for f-string (python 3.6): https://peps.python.org/pep-0498/

Reply:
done